lib/runtime-core/src/codegen.rs

```diff
@@ -43,6 +43,8 @@ pub trait ModuleCodeGenerator<FCG: FunctionCodeGenerator<E>, RM: RunnableModule,
 
     /// Adds an import function.
     fn feed_import_function(&mut self) -> Result<(), E>;
+
+    fn feed_compiler_config(&mut self, config: &CompilerConfig) -> Result<(), E> { Ok(()) }
```

Comment:
I think it would be nice if the trait methods were in the order that the parser calls them. So this one would be higher.